app/views/comments/_login_cta_comment.html.erb

```diff
@@ -12,7 +12,7 @@
         </p>
```

Comment:
I couldn't find a reference to this file anywhere in the codebase 👀, it seems like we may not be using it?

Reply:
yeah, it looks like it can be deleted